Makefile

```diff
@@ -1,4 +1,11 @@
-SHELL := /bin/bash
+# Setting SHELL to bash allows bash commands to be executed by recipes.
+# Options are set to exit when a recipe line exits non-zero or a piped command fails.
+SHELL := /bin/bash -o pipefail
```

Comment:
Should we still enforce `/bin/bash` or use `/usr/bin/env bash` as in operator-sdk Makefile?

Reply:
https://github.com/operator-framework/operator-sdk/blob/0a7bfd8e41b8ac75e18d7138690ac9bddd6ea49a/testdata/go/v3/memcached-operator/Makefile#L64

Reply:
`/usr/bin/env bash` is more portable, basically. It's usually fair to assume that `env` exists at `/usr/bin`, but in some distros or operating systems, `bash` may not be at `/usr/bin/bash`, perhaps at `/usr/local/bin/bash` and without any symlink for example. Same reason you'll often see `.py` files starting with ex. `#!/usr/bin/env python2`.

Reply:
Make sense! I will set it to use `env` for finding bash.